mne_bids/tests/test_mne_bids.py

```diff
@@ -368,3 +368,13 @@ def test_set():
 
     cmd = ['bids-validator', '--bep010', output_path]
     run_subprocess(cmd, shell=shell)
+
+
+def test_assert_mne_version():
```

Comment:
is the test ok her, or does it have to move somewhere else?

Reply:
Not sure if we need a test at all for this (because `check_version` is tested in MNE), but I personally wouldn't mind having it.

Though I would put it into one of the other test functions instead of making a new test function. For example into `test_set` or `test_fif` and then make it small and unobtrusive with a small comment.

The general test file is correct

Reply:
This also relates to your second 2. --> I think for this code addition it is fine to recycle the existing tests